### DISCUSSION TITLE
Support of ReferenceLinkBaseRef for location

### DISCUSSION THREAD

**Discussion by bmaisonn** [manusimidt/py-xbrl#79] [Q&A] [answered]
Hi everyone,
We started to use that library to parse XBRL documents from FERC! That's a great project!. We noticed that some link base types are not implemented yet, especially the one related to location
`
xlink:href="ferc-core-ref-form-location_2021-01-01_ref.xml" xlink:role="http://www.xbrl.org/2003/role/referenceLinkbaseRef"
`
Is it correct ?

**Comment by manusimidt** [comment, marked as answer]
Hey, 

Yes, that's correct. The reason why I developed py-xbrl in my free time was that I wanted to parse 10-Q and 10-K forms from SEC Edgar (annual/quarterly financial reports of publicly traded companies).
Therefore py-xbrl currently can only parse 'calculation', 'definition', 'presentation' and 'label' linkbases. 

I never tried to parse data from the FERC before, but it sounds very interesting. Which form type are you trying to parse? Are you using the data from this page? https://www.ferc.gov/filing-forms/eforms-refresh/migrated-data-downloads

I would like to try py-xbrl with the FERC filings, but I don't quite understand the structure of the data and the different form types yet.

**Reply by bmaisonn** [reply]
Thx for your reply. That's indeed the type of documents that we try to parse.
